src/helpers/discord_utils.ts

```diff
@@ -475,6 +475,22 @@ export async function sendEndRoundMessage(messageContext: MessageContext,
     }
 
     const songAndArtist = bold(`"${gameRound.originalSongName}" - ${gameRound.artistName}`);
+    if (isMultipleChoiceMode) {
+        await gameRound.interactionMessage.edit({
+            embeds: [{
```

Comment:
Re-use the embed in both cases

Reply:
I don't think I can because of the differing thumbnailUrl and footerText fields